build.gradle

```diff
@@ -1,53 +1,39 @@
 apply plugin: 'idea'
 
-version '1.0.0-SNAPSHOT'
-
 subprojects {
-    apply plugin: 'idea'
-    apply plugin: 'java'
-//    apply plugin: 'checkstyle'
-//    apply plugin: 'findbugs'
-//    apply plugin: 'jacoco'
-//    apply plugin: 'pmd'
+
+    buildscript {
+        repositories {
+            mavenCentral()
+            mavenLocal()
```

Comment:
`mavenLocal()` should be listed before `mavenCentral()` as Gradle checks the repositories in definied order and stops at the first repository which contains the requested artifact.